cmd/speedtest.go

```diff
@@ -274,6 +280,12 @@ func driveSpeedTest(ctx context.Context, opts madmin.DriveSpeedTestOpts) madmin.
 				}
 				results = append(results, result)
 			}
+			for _, inp := range ignoredPaths {
+				results = append(results, madmin.DrivePerf{
+					Path:  inp,
+					Error: errFaultyDisk.Error(),
```

Comment:
Do we need errSkipPath for this? This err is confusing.

Reply:
No the error is correct @jiuker since we need to tell that drive is faulty so its ignored.